.github/codecov.yml

```diff
@@ -1,7 +1,17 @@
+# https://docs.codecov.io/docs/commit-status
+coverage:
+  status:
+    project:
+      default:
+        informational: true
```

Comment:
This will keep the status green, even if we drop coverage %, so it's less annoying.

![image](https://user-images.githubusercontent.com/567105/118983136-053a8c00-b974-11eb-9ed9-d902bad5de6d.png)